lib/logstash/filters/xml.rb

```diff
@@ -35,29 +35,30 @@ class LogStash::Filters::Xml < LogStash::Filters::Base
   #
   # XML in the value of the source field will be expanded into a
   # datastructure in the `target` field.
-  # Note: if the `target` field already exists, it will be overridden
-  # Required
+  # Note: if the `target` field already exists, it will be overridden.
+  # Required.
   config :target, :validate => :string
```

Comment:
Do you want me to amend my commit to state that the `target` is only mandatory when `store_xml` is true? If not, perhaps we can complete the review and merge the PR?